tiled/client/node.py

```diff
@@ -543,6 +549,87 @@ def _ipython_key_completions_(self):
             # Do not print messy traceback from thread. Just fail silently.
             return []
 
+    def create_array(self, array, metadata=None, specs=None):
```

Comment:
On second thought, I think `write_array` (matching `read()`) would be a better name here. From the user's point of view, they aren't _creating_ the array (they already have the array) they are writing it into tiled's storage.